src/foam/comics/v2/DAOControllerConfig.js

```diff
@@ -61,6 +61,21 @@ foam.CLASS({
       name: 'browseSubtitle',
       expression: function(of) { return "View all " + of.model_.plural.toLowerCase() +'.' }
     },
+    {
+      class: 'foam.u2.ViewSpecWithJava',
```

Comment:
@kgrgreer RE: you comment on my closed pr (https://github.com/foam-framework/foam2/pull/3481) about changing this to ViewSpec and then making ViewSpecWithJava a refinement on ViewSpec. Was thinking I should inlcude the change after this PR is done in a separate PR so that this PR doesn't get too bogged down

Reply:
Okay, then at least add a TODO.

Reply:
Done.